### ISSUE TITLE
oelint.var.order shouldn't warn on append/remove operations

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#146]
This for example is intentionally done as otherwise use cases like update check aren't working
```
PYPI_PACKAGE = "foo"

inherit pypi
inherit setuptools3

SRC_URI += "file://foo-super.patch"
```

**Comment by priv-kweihmann** [comment]
No that's not right. It's better to advertise `SRC_URI_append` only

**Comment by priv-kweihmann** [comment]
Superseded by #147
